mu4/userscores/qml/MuseScore/UserScores/UserScoresContent.qml

```diff
@@ -3,6 +3,8 @@ import QtQuick.Layouts 1.3
 import MuseScore.UiComponents 1.0
 import MuseScore.UserScores 1.0
 
+import "internal"
```

Comment:
What is the motivation of the import?
It doesn't seem to be used in the file

Reply:
Needed for RecentScoresView. Line 70